core/src/main/java/org/mule/DefaultMuleEvent.java

```diff
@@ -69,6 +70,8 @@
 
     private static Log logger = LogFactory.getLog(DefaultMuleEvent.class);
 
+    public static final int DEFAULT_LENGTH_MULE_EVENT_ID = UUID_LENGTH + 2 ;
```

Comment:
By default It's a zero. However, It can have a customizable value. 
I'm checking that [here](https://github.com/mulesoft/mule/pull/4324/files#diff-d43440932499749e49e2c518b38914efR219)